PlaidLink/Classes/PLDLinkBankMFAContainerViewController.m

```diff
@@ -59,9 +70,16 @@ - (void)viewDidLoad {
   self.navigationItem.backBarButtonItem.target = self;
   self.navigationController.interactivePopGestureRecognizer.delegate = self;
 
-  PLDLinkBankMFALoginViewController *viewController =
-      [[PLDLinkBankMFALoginViewController alloc] initWithInstitution:_institution
-                                                          product:_product];
+  PLDLinkBankMFALoginViewController *viewController;
```

Comment:
with options as a property you can now just pass the variable to the next view controller